decode_test.go

```diff
@@ -559,6 +559,17 @@ var unmarshalTests = []struct {
 		"a: []",
 		&struct{ A []int }{[]int{}},
 	},
+
+	// Line number information from parser
+	{
+		"\n\n\n\n\n\n\n\n\n\na: one\nb: two\nc: three\n",
+		&struct {
+			A    string
+			B    string
+			Line int ",linenum"
+			Col  int ",colnum"
+		}{"one", "two", 11, 1},
```

Comment:
It's not clear what the line number and column number information mean here. There is a struct with multiple fields, and only a single exposed number information which maps to the first field apparently. What's are the semantics, and how is it useful?